### PR TITLE
Homework to review

### PR DESCRIPTION
**Pull request by MaximilianLipkevich**
Homework lesson 13, 15, 16

### CODE REVIEW

File: Homework Lesson 16/Task 3.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    def __init__(self):
+        self.products = {}
+        self.income = 0
+
+    def add(self, product, amount):
+        if product.name not in self.products:
+            self.products[product.name] = {
+                'product': product,
+                'amount': amount,
+                'price': product.price * 1.3
```
Comment (Oleksandr139Kotvytskyi) [inline Homework Lesson 16/Task 3.py:17]: why 1.3?)

**Review by Oleksandr139Kotvytskyi — APPROVED** [review]
LGTM

File: Homework Lesson 16/Task 3.py (hunk 1 above)
Comment (MaximilianLipkevich) [reply Homework Lesson 16/Task 3.py:17]: The task says "add(product, amount) - adds a specified quantity of a single product with a predefined price premium for your store(30 percent)"

**Review by Oleksandr139Kotvytskyi — APPROVED** [review]
LGTM but why you need this param Book.total += 1?

**Pull request merged**